src/net/CUDPConnection.cpp

```diff
@@ -774,6 +795,9 @@ void CUDPConnection::FreshClient(ip_addr remoteHost, port_num remotePort, uint16
 
     ipAddr = remoteHost;
     port = remotePort;
+
+    //IPaddress addr = { remoteHost, remotePort };
```

Comment:
i think we can remove this now... looks like this happens in `AvaraTCP::CheckSockets()`